docs/ml-clustering.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: global
+title: Clustering - ML
+displayTitle: <a href="ml-guide.html">ML</a> - Clustering
+---
+
+In this section, we introduce the corresponding pipeline API for
```

Comment:
I thought this was supposed to read "In this section, we introduce the pipeline API for clustering in mllib"? The pipelines API doesn't correspond to `mllib` since it's missing a lot of features